src/Test/Lint.hs

```diff
@@ -94,11 +94,11 @@ main = shaken test $ \args obj -> do
         need [obj "tracker-source.c", obj "tracker-source.h"]
         cmd "gcc" ["-c", obj "tracker-source.c", "-o", out]
     where gen t f = unit $ if isWindows
-                           then cmd "cmd /c" ["echo" ++ t ++ " > " ++ f]
-                           else cmd Shell "echo" ["x", ">", f]
+                           then cmd "cmd /c" ["echo " ++ t ++ " >" ++ f]
+                           else cmd "sh -c" ["echo " ++ t ++ " >" ++ f]
```

Comment:
I don't want Shell to disappear, it's quite handy, since it's a built in part of the Haskell process library, and can be quite useful, and avoids an isWindows. But reworking it so I do either `sh -c` or `cmd /c` in a portable way seems valuable - so instead of passing `Shell` to Haskell to deal with, I do it myself, then `fsatrace` should work fine. I've ticketed it here: https://github.com/ndmitchell/shake/issues/308